packages/taxdat/R/stan_input_helpers.R

```diff
@@ -1652,6 +1656,7 @@ impute_adm0_obs_single <- function(sf_cases_resized,
             if (nrow(x)> 1) {
               dplyr::rowwise(x) %>% 
                 # Only compute tfrac-adjusted counts for full obs
+                ## ECL: So this is assuming a set_tfrac = TRUE config, right?  
```

Comment:
I think it's fine for the imputation but I wanted to understand the interpretation

Reply:
This is a good point. Here it will use the value of set_tfrac, as this happens before imputation, so if set_tfrac is true then everything will be the same. If set_tfrac is false then it will actually use the tfrac values.